scripts/etl/schedulers/main.py

```diff
@@ -9,13 +9,16 @@
 from __future__ import division
 from __future__ import unicode_literals
 
+import logging

```

Comment:
Feel free to land changes in directories under `scripts` without need for review (e.g. scripts/foo/*)